docs/application/web/guides/data/file-system.md

```diff
@@ -8,36 +11,32 @@ The Filesystem API provides access to accessible parts of the file system, which
   - You can [read or write to a file](#managing-files-and-directories).
   - You can [copy and move files and directories](#managing-files-and-directories) within the virtual file system.
-  - You can [delete files and directories](#creating-and-deleting-files-and-directories) from the virtual file system.
 
 ## File and Directory Access
 
-You can access the virtual file system using the `FileSystemManager` interface (in [mobile](../../../../org.tizen.web.apireference/html/device_api/mobile/tizen/filesystem.html#FileSystemManager), [wearable](../../../../org.tizen.web.apireference/html/device_api/wearable/tizen/filesystem.html#FileSystemManager), and [TV](../../../../org.tizen.web.apireference/html/device_api/tv/tizen/filesystem.html#FileSystemManager) applications):
-
-- To access a file or directory within the virtual file system, you must use the fully qualified path, `<root name>/<path>`, where `<rootname>` is the name of the virtual root and `<path>` is the relative path to the file or directory within the root.
-
-> **Note**
-> When you use a path to access the device file system, make sure that the file path encoding uses the default encoding of the platform.
+You can access the virtual file system using the `FileSystemManager` interface (in [mobile](../../api/latest/device_api/mobile/tizen/filesystem.html#FileSystemManager), [wearable](../../api/latest/device_api/wearable/tizen/filesystem.html#FileSystemManager), and [TV](../../api/latest/device_api/tv/tizen/filesystem.html#FileSystemManager) applications):
 
-- To access a file or directory, you must also retrieve a file handle using the `resolve()` method of the `FileSystemManager` interface.A file handle is a reference object that points to and represents a file or directory.
+- To access a file or directory within the virtual file system, you can use the fully qualified path, `<rootname>/<path>`, where `<rootname>` is the name of the virtual root and `<path>` is the relative path to the file or directory within the root. Other option is to use absolute path to a file located in target device memory. Regardless of virtual root usage, rights to access file location as well as rights to read and write to a file are needed when applicable.
 
-The `isFile` and `isDirectory` attributes of the `File` interface (in [mobile](../../../../org.tizen.web.apireference/html/device_api/mobile/tizen/filesystem.html#File), [wearable](../../../../org.tizen.web.apireference/html/device_api/wearable/tizen/filesystem.html#File), and [TV](../../../../org.tizen.web.apireference/html/device_api/tv/tizen/filesystem.html#File) applications) identify the type of the object: for example, for a file, the `isFile` attribute is set to `true` and the `isDirectory` attribute to `false`.
+  > **Note**  
+  > When you use a path to access the device file system, make sure that the file path encoding uses the default encoding of the platform.
```

Comment:
Please rephrase to:

When you use a path to access the device file system, ensure that the file path encoding uses the default encoding of the platform.

Reply:
Done